internal/webserver/server.go

```diff
@@ -87,6 +87,11 @@ func (webserver *WebServer) ConfigureStandardRoutes() {
 
 	router.Use(handlers.ProcessCORS(webserver.config.Service.CORSConfiguration))
 
+	// Handle the CORS preflight request
+	router.Methods(http.MethodOptions).MatcherFunc(func(r *http.Request, rm *mux.RouteMatch) bool {
+		return r.Header.Get(handlers.AccessControlRequestMethod) != ""
```

Comment:
should we do allowable list validation instead of everything except empty?

Reply:
That is handled by `HandlePreflight` which sets the `AccessControlAllowMethods` in the header